queries.sql

```diff
@@ -9,4 +9,45 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Please add the `SELECT * FROM animals;` query before `ROLLBACK` because you will verify whether that change was made or not so kindly add this query :slightly_smiling_face: